packages/admin/resources/views/components/notification-manager.blade.php

```diff
@@ -1,8 +1,8 @@
 <div
     x-data="{
-        notifications: {{ json_encode(session()->pull('notifications', [])) }},
-        add: function (event) {
-            this.notifications.push(event.detail)
+        notifications: {{ Js::from(session()->pull('notifications', [])) }},
```

Comment:
Can we swap this for `@js`? Or, use the fully qualified class name, I don't like aliases

Reply:
Yeah sure.